jujugui/static/gui/src/app/assets/svgs/card-american-express.svg

```diff
@@ -0,0 +1,19 @@
Vlv0euLhEX6pK5SXTcQ7E844JD6Mvm+9CHKYes9oTwrgyTP9t/xPtU9D+EfsSZoXx/9MPtSgBOh09ZTwbKbIGFrECvq1P44d29vKB+EdKZ65e9bA0K9+g9e7eDLqwxp0hG3oE+jFyfecJjQ3/H4lPQ5yhwnxsCyAdX99gVfiD6EUxNAI+vz6CD1SVBb7OiyKBZQABO9+JaN826SqGv7Rfh+81r3MOO1p7QP6Ax3+sfOOlYW/bFgQeCHfpgfnbhhwsiFUHenFLNQA5uf/ux6EdMVwnQQ+4jv0iJrg7O+4DemhWzR6iuDyG2uIflftQh7tEloLPkb8P4KOhHJPI3PW2BSORRxE9QQnmaGraOzGn38nPrtGjQ7A85+SiS5xJAaiL6dKsLxLPp4bpn9LXHM603+fdxD0flATc43w4vOj2Ya+hD0p90o2lPqF710bWdmQ7Ep6AfZvH/Z449EO5O08bh9jOgHwG9mwCHMbXGdkma1E/KcNAjRJ3vQfd6n8zNAY//fHoryjFkQl9fKvnLzei+hL56U/Au9CH8J/o5Y7p/bgLIzR+FuAN6jMUrHCDFBxtH+f3TDqz9W0Ef41XtV/KDyd3stV86QF0bKeinrpUAWPv2n8J4ytQfVfgC9BiSij3o+7j3ZNC8aJG08AX6OtLwMReKfuCJpmwlsd1gPUiKe9UPI3aIYlfZiyBz5NPMg3v3u1X4qdCPCFcCoFGg8K+aD2WYGLgfb9B00EfvETEX9f4a92F5mvo+0M+x4DauDn10fGf0c+CaBwSArH7nn25RWZmjfO5zJUABAcIqQPnj5kMZCGaoDBaca9BD4FncQ0Vro/UuuIeMivrHVv0cEW7r6uj/EOjDgReI5a9RqP0/G/oR40oAR0D8m62OA+j5wP6fLSs61hlnGQ6YPt4hM8szm1iwDnp06Xy/rfcQEO5Bfzb04aLjj8cMdPE6gz56FV7QKT2zgqoP7t31VoGWC/qZ+T+w9jPUlQAazK8B6sPi9nyg8llWaBY+Iz3OIC6nHevsKoMCU0OeAvFJ0CdB9rh/N9zDn7YF+t6e9dFV4h/C/Mi7Qx+MI8TXEh/5pn9Jbr13jZ8Z/Yh2JQADxb9xuAys5wMHeZYV7OUSPgRrn6+bnDOUU71jnfwyNDA56m2lR69AD3rhhbbm/Wa9h7hgfXbacRmafxn6Yco2NIc4ut4s/LDQZ5IBl/tPjn5EuxKAINCvOPmVbc8H6CAlcD8DGf+NhdcALdP0cvMsBhrcLpIjHmId1gB98mOMvTf68triHj0Xp3z0fn/ow6mj/33ow+B2YsEvl9DvTlkFfvjJR6GuBBCLRMkHMC9SQrpn24UEPpy4XowCd3jfYvpOpWfk93EP+RdKPrQ+quqHqQx6HusPQz92pdyN2/VsuwHYL4d+9Ar9Z2N0C1va538r8CzTEmD+k9cE+mwtg2cpQQn+F1HvT9Cy+xEUl5OWGOek99a9zEPiAI2mUSbd61mv99DuuA+mHUjoQWJyeAaLs1n1URf7jm/Y95i968yjQnLiLAyXCV/Z/kOgH5FaAsxxDABNUOjna2cpQT2dnaaZ09+9MVUkdkH4AruYQ5z8GaxLLZpwR1uK1/LUdNw7gLagh4oD+uKoUyTp6wyIZ5grM+PoV9ePKvwYUUG/Cj+6zkbKebi4+3JciD3btUmAYkKg8Tp5kRVSX4piJdFT5di/WndguqRpfOq498krsrVZpthBD9HviXu4ewH60CL6PxD6F5Ggq1z0jonyqfuZ0Y/4VwJw1vh0UgbWmwteR3ApMaTS9w11vUColkt3RZKsYziSOiVeAz3MvVPvoX5RCN+HftpfM9H2hNPZ6B1nwRRJQb+4/snRj2hXAiBr8WOgAghI3EwJzkiBYzDXQlDkrfubxvroEI2XKzTPKj26HPRoXp9zujw4L+A+tPB/u/zAAzZ7NbQyHO+98G7mB3kT+gqgHHvAF/qf8lsi0bgK/6OaKwEwgL+1B7X3U+KjAn3BznbuCj5g9gLxdOq4F+jRVYCoCF1ezAsQXEDtDvThgiMtQ3sZ+jB4EZJGRIJeCvoFfchcDLyY6s3tCnaxdzgrAWClQOFvEX2tuduUYARP7RXvBF10L6apx9+HWayx6SB20KP3o3APU2c4K7jvTiXwEPrQfQqCZyHBTrm20IfMHwj9iPaQAGWEJR/Qu00Jal0kRjH7Gc0t0Omoj2IbgCMeAjdBD8miSOPXmLsAmZCtIEu+SeAC+tCdvTJzi7gIrOj/a6Afg1oJ4At59j3uFkzIClwXEMz+ww3ZcmjvGk8ZpIFteDvbwfPxUuZN0MPINe4hcIEwIZvB4P5zQj+GmSFytn2ZOKU4CDycB41xS1xs6Vv5d5grAbT8/9HAcZYPdHwTdswTqvis3Yn+potrUw8RD/WCuWKwW6DAw/W+wD0sFOj3GCQw6/rhXNp6S9QPmtexFeWH0I/hHKIrBh40vyf0GcpKAM77b1+/KhMo0fOB/Ous6AP9EBB3s1vOGVLL0KDb0VYMnpmC2MOVvsaWgCuPPRjJbKEfMaSycPMwJPkicR1hEf7XQz8GuBKAoy1r0POBYmdZoSl7Nj2keJO4wCUtdKyTXwa4dXdh/A7CrlElTLvrHlVDdq2rTcDtPaavgyz619CH8B/r2OOjqwngfaD7woB5lhXURW7gusAQxT7kfoZyGt8Gv/V7He0d0MPsQ0h16PcIJWNF/YOhfydUn6Vr9PPV380pcrNOa7DO/D70SoACgosS3peNsSIxQFyD8jNGdRbPha8y2C55f0VfwD3c9ZgfQn8n0AN/wHkYres79MH3JzdO4B8d/RjUSgAfOegOkYuUoG5f1GLzRzX7WLaRfCDoYV94dV/bKZKkCqFDzU2ZgFu9RT8FfTm9eNUTMnVzuhWJhDQccb4/8fVb+6rr+wfxp8c/Z+BHzYCqz48K4E+/f87Aj5yBPxPg/28gQ3/U7gEPAQCA02jH05wGWgAAAABJRU5ErkJggg=="></image>
```

Comment:
Spencer sourced them for me, he said that he couldn't find svgs for all of the logos. So the problem we have is that our spriting system only handles svgs and we have svg and png logos, I guess I could have a map of brand names to files and load the images from the server and resize them...